docs/user/04-metrics.md

```diff
@@ -313,14 +313,16 @@ spec:
 
 The Metric agent is configured with a generic scrape configuration, which uses annotations to specify the endpoints to scrape in the cluster.
 
-For metrics ingestion to start automatically, simply apply the following annotations either to a Service that resolves your metrics port, or directly to the Pod:
-
-| Annotation Key                     | Example Values    | Default Value | Description                                                                                                                                                                                                                                                                                                                                 |
-|------------------------------------|-------------------|-------------- |---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `prometheus.io/scrape` (mandatory) | `true`, `false` | none | Controls whether Prometheus automatically scrapes metrics from this target.                                                                                                                                                                                                                                                             |
-| `prometheus.io/port` (mandatory)   | `8080`, `9100` | none | Specifies the port where the metrics are exposed.                                                                                                                                                                                                                                                                                           |
-| `prometheus.io/path`               | `/metrics`, `/custom_metrics` | `/metrics` | Defines the HTTP path where Prometheus can find metrics data.                                                                                                                                                                                                                                                                               |
-| `prometheus.io/scheme`             | `http`, `https` | If Istio is active, `https` is supported; otherwise, only `http` is available. The default scheme is `http` unless an Istio sidecar is present, denoted by the label `security.istio.io/tlsMode=istio`, in which case `https` becomes the default. | Determines the protocol used for scraping metrics — either HTTPS with mTLS or plain HTTP. |
+For metrics ingestion to start automatically: 
```

Comment:
```suggestion
For metrics ingestion to start automatically, use the annotations of the following table. If Istio is active (recommended), apply them to a Service that resolves your metrics port. Only if Istio is inactive, you can alternatively apply the annotations directly to the Pod.
```
I assume that for most users, Istio is going to be active?
Or: What do you mean with "Istio is active" - the Istio module is added to the cluster (by default, yes)? Or Istio sidecar injection is enabled (by default, nope)?

Reply:
with "istio is active", I meant the Istio sidecar is present. I will change the wording there to make it more clear